notification/docs/IntegrationGuide.md

```diff
@@ -0,0 +1,65 @@
+# Integration of payment into checkout process
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+- [Deployment](#deployment)
+  - [Glossary](#glossary)
+  - [Requirements](#requirements)
+- [Configuration](#configuration)
+      - [Environment variables to configure the notification module:](#environment-variables-to-configure-the-notification-module)
+  - [After deployment](#after-deployment)
+    - [Register the endpoint](#register-the-endpoint)
+- [Note](#note)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Deployment
+
+### Glossary
+In this process, there are 2 parties involved:
+
+**Adyen** - the payment provider which will send notifications
+**Notification module** - hosted service that receives notifications from Adyen,
+processes and stores them on a commercetools platform project.
+
+### Requirements 
+Node.js version LTS 8 or higher is supported.
+
+## Configuration
+
+##### Environment variables to configure the notification module:
+Name | Content | Required | Default value
+------------ | ------------- | ------------- | -------------
+CTP_PROJECT_KEY | commercetools project key (you can get in the commercetools Merchant Center) | **YES** |
+CTP_CLIENT_ID | commercetools client ID (you can get in the commercetools Merchant Center) | **YES** |
+CTP_CLIENT_SECRET | commercetools client secret (you can get in the commercetools Merchant Center) | **YES** |
+LOG_LEVEL | bunyan log level (`trace`, `debug`, `info`, `warn`, `error`, `fatal`)| NO | `info`
+PORT | port on which the application will run | NO | 443
+
+Check out the deployment [Best Practices documentation](../../docs/BEST_PRACTICES.md)
+
+### After deployment
+
+After deployment and getting a static URL to notification module
```

Comment:
for here, IMO we should use the same terms for static URL or endpoint URL, I would use notification endpoint URL. And what is getting a static URL means for the users? 😕

Reply:
Is it better?
```
After deployment you have to register the Notification module public URL in the Adyen Customer Area in order to receive notifications.
```

Reply:
yes, better 👍